src/app/pages/ordenesdepago/components/set-movimientocontable/set-movimientocontable.component.ts

```diff
@@ -155,7 +167,11 @@ export class SetMovimientocontableComponent implements OnInit, OnDestroy {
                 this.cuentasContablesConcepto.push({cuenta: this.cuentaCredito});
                 if (this.cuentasContablesConcepto.length === total) {
                   this.boolean = true;
-                  if (this.tituloAccion === 'ver') this.ordenesPago();
+                  if (this.tituloAccion === 'ver') {
+                    this.movimientoContable.patchValue({
+                      cuentaCredito: (this.cuentasContablesConcepto[this.cuentasContablesConcepto.findIndex((e: any) => e.cuenta.Codigo === this.ordenPago.CuentaValorNeto)]),
```

Comment:
Linea muy larga... Dividirla por legibilidad. Una forma podría ser esta:
```suggestion
                      cuentaCredito: (this.cuentasContablesConcepto[
                        this.cuentasContablesConcepto.findIndex(
                          (e: any) => e.cuenta.Codigo === this.ordenPago.CuentaValorNeto
                        )
                      ]),
```

Por otro lado, por qué no usar no mas ... ?
```suggestion
                      cuentaCredito: this.cuentasContablesConcepto
                        .find((e: any) => e.cuenta.Codigo === this.ordenPago.CuentaValorNeto),
```